helm/rook-operator/charts/ceph-cluster/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.global.rookEnabled }}
```

Comment:
all subchart resources are disabled unless the value is set to true in giantswarm/config